.changes/nextrelease/feature-doctype.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "type": "feature",
+    "category": "",
+    "description": "Enable support for internal document model types"
```

Comment:
Copy/pasted below so that you don't have to click into the diff to see it:
"Enable support for internal document model types to allow the sending of raw json directly to services".